git_theta/file_io.py

```diff
@@ -0,0 +1,130 @@
+import tensorstore as ts
```

Comment:
This file was deleted in #114. Probably got added back into this branch inadvertently.

Reply:
I originally prototyped in this file and it got added back. Deleted